to_do_app/lib/addTask/form_to_add_new_task.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+
+class FormToAddNewTask extends StatelessWidget {
+  FormToAddNewTask({
+    Key? key,
+    required this.titleController,
+    required this.descriptionController,
+  }) : super(key: key);
+
+  final TextEditingController titleController;
+  final TextEditingController descriptionController;
+
+  @override
+  Widget build(BuildContext context) => Form(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          TextFormField(
+            decoration: const InputDecoration(
+              border: UnderlineInputBorder(),
+              hintText: 'Task title',
+            ),
+            validator: (value) {
+              return (value == null || value.isEmpty)
+                  ? 'Please enter some text'
+                  : null;
+            },
+            controller: titleController,
+          ),
+          TextFormField(
+            decoration: const InputDecoration(
+              border: UnderlineInputBorder(),
+              hintText: 'Task description',
+              labelStyle: TextStyle(fontSize: 20),
+            ),
+            validator: (value) {
+              return (value == null || value.isEmpty)
+                  ? 'Please enter some text'
+                  : null;
+            },
```

Comment:
Same here.
Also you can create an extension for `String` to check wether it's null or empty as you use it multiple times